bin/dev_hdd0/home/00000001/localusername

```diff
@@ -1 +0,0 @@
-User
```

Comment:
I wouldn't deleted this file.

Reply:
no, you can, rpcs3 create it by defaut if this file is not here, and you can also change pseudo here